classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -872,14 +872,14 @@ function fetchTemplateBody($args, $request) {
 		$dispatcher = $request->getDispatcher();
 		$context = $request->getContext();
 
-		$template->assignParams(array(
+		$template->assignParams([
 			'contextUrl' => $dispatcher->url($request, ROUTE_PAGE, $context->getPath()),
 			'editorialContactSignature' => $user->getContactSignature(),
-			'signatureFullName' => $user->getFullname(),
+			'signatureFullName' => htmlspecialchars($user->getFullname()),
 			'passwordResetUrl' => $dispatcher->url($request, ROUTE_PAGE, $context->getPath(), 'login', 'lostPassword'),
 			'messageToReviewer' => __('reviewer.step1.requestBoilerplate'),
-			'abstractTermIfEnabled' => ($this->getSubmission()->getLocalizedAbstract() == '' ? '' : __('common.abstract')), // Deprecated; for OJS 2.x templates
-		));
+			'abstractTermIfEnabled' => htmlspecialchars($this->getSubmission()->getLocalizedAbstract() == '' ? '' : __('common.abstract')), // Deprecated; for OJS 2.x templates
```

Comment:
I think this isn't the abstract but just a locale key for the word "Abstract".

Reply:
You're right; removed the `htmlspecialchars` call.